docs/source/installed_packages.rst

```diff
@@ -78,6 +78,11 @@ the ``setup.py`` might look like
 
    If you use :doc:`setuptools <setuptools:index>`, you must pass the option ``zip_safe=False`` to
    ``setup()``, or mypy will not be able to find the installed package.
+   
+.. note::
+
+   If you use `pyproject.toml` instead of `setup.py`, you don't need to do anything specific. 
+   `py.typed` will be picked up automatically.
```

Comment:
This is incorrect. What happens with pyproject.toml depends entirely on what build backend (e.g. setuptools) you are using.

Reply:
Oh I see. Thank you for pointing it out then!